File: app/src/main/java/ua/edu/deanoffice/mobile/studentchdtu/service/client/interfaces/GetApplication.java
package ua.edu.deanoffice.mobile.studentchdtu.service.client.interfaces;

import okhttp3.ResponseBody;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface GetApplication {
    @GET("generate-application")
    Call<ResponseBody> getRequest(@Query("applicationType") int id, @Query("json") String json);
}

File: app/src/main/java/ua/edu/chdtu/deanoffice/mobile/service/client/requests/interfaces/GetApplication.kt
package ua.edu.chdtu.deanoffice.mobile.service.client.requests.interfaces

import okhttp3.ResponseBody
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface GetApplication {
    @GET("param")
    fun getRequest(@Query("applicationType") id: Int, @Query("json") json: String): Call<ResponseBody>
}